tsconfig.json

```diff
@@ -11,6 +11,6 @@
     "noUncheckedIndexedAccess": true,
     "strict": true,
     "esModuleInterop": true,
-    "resolveJsonModule": true,
+    "resolveJsonModule": true
```

Comment:
🔧  I think we should keep trailing commas, they make future diffs more compact and easier to read.

Reply:
Hmm this is done automatically when I ran prettier